libraries/rib-base/src/main/java/com/badoo/ribs/android/integrationpoint/IntegrationPoint.kt

```diff
@@ -108,7 +130,8 @@ abstract class IntegrationPoint(
     abstract fun handleUpNavigation()
 
     override fun show(dialog: Dialog<*>, onClose: () -> Unit) {
-        dialogs[dialog] = dialog.toAlertDialog(rootViewHost.context, onClose).also {
+        val context = rootViewHost?.context ?: return
```

Comment:
I don't think we should return here silently

Reply:
Added `RIBs.errorHandler.handleFatalError`.